src/main/scala/mesosphere/marathon/core/storage/repository/impl/DeploymentRepositoryImpl.scala

```diff
@@ -57,32 +65,47 @@ object StoredPlan {
   }
 
   def apply(proto: Protos.DeploymentPlanDefinition): StoredPlan = {
-    val version = if (proto.hasTimestamp()) {
-      OffsetDateTime.parse(proto.getTimestamp, DateFormat)
+    val version = if (proto.hasTimestamp) {
+      Try(OffsetDateTime.parse(proto.getTimestamp, DateFormat)).getOrElse(OffsetDateTime.MIN)
     } else {
       OffsetDateTime.MIN
     }
     StoredPlan(
       proto.getId,
-      OffsetDateTime.parse(proto.getOriginalRootVersion, DateFormat),
-      OffsetDateTime.parse(proto.getTargetRootVersion, DateFormat),
+      Try(OffsetDateTime.parse(proto.getOriginalRootVersion, DateFormat)).getOrElse(OffsetDateTime.MIN),
+      Try(OffsetDateTime.parse(proto.getTargetRootVersion, DateFormat)).getOrElse(OffsetDateTime.MIN),
```

Comment:
After migrating the old DeploymentPlans those values should be always available?
If we would run into this: taking `OffsetDateTime.MIN` would hide the problem?